pkg/cmd/server/api/helpers.go

```diff
@@ -305,6 +305,7 @@ func GetKubeClient(kubeConfigFile string) (*kclient.Client, *restclient.Config,
 	// TODO: this should be configured by the caller, not in this method.
 	kubeConfig.QPS = 100.0
 	kubeConfig.Burst = 200
+	kubeConfig.ContentType = "application/vnd.kubernetes.protobuf"
```

Comment:
This needs to fallback to json to handle a rolling upgrade case.  These functions are used when wiring up controllers, so a new controller to old master case needs to send and accept json.